keyboards/kiwikey/kawii9/rev2/info.json

```
@@ -0,0 +1,86 @@
+{
+    "manufacturer": "KiwiKey",
+    "keyboard_name": "Kawii9 Rev.2",
+    "maintainer": "KiwiKey",
+    "url": "https://kiwikey.vn/mechkey/kawii9/",
+    "bootloader": "atmel-dfu",
+    "processor": "atmega32u4",
+    "usb": {
+        "device_version": "2.0.0",
+        "pid": "0x3302",
+        "vid": "0x4B57"
+    },
+    "build": {
+        "lto": true
+    },
+    "features": {
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": false,
+        "rgblight": false,
+        "rgb_matrix": true,
+        "encoder": true
+    },
+    "bootmagic": {
+        "enabled": true,
+        "matrix": [0, 0]
+    },
+    "debounce": 5,
```

1. debounce default is 5
2. [0, 0] is default for bootmagic
3. bootmagic moved to features
```suggestion
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": false,
        "nkro": false,
        "rgblight": false,
        "rgb_matrix": true,
        "encoder": true
    },
```